### PR TITLE
Adding numerals for critter appraisals

### PR DESCRIPTION
**Pull request by izawwlgood**
May be missing a value, and may have 'mythical' difficulty text incorrect. Will double check.

### CODE REVIEW

**Comment by izawwlgood** [comment]
I do think we may be able to simplify the string match to simply everything after the 'is a(n)...'. But either way!

File: textsubs.lic [hunk 1]
```diff
@@ -36,7 +36,32 @@ before_dying do
 end
 
 # defaults
-
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is a creature completely beneath your notice.', 'Taking stock of its \1 abilities, \2 is a creature completely beneath your notice. (-13)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is something you could wipe the floor with.', 'Taking stock of its \1 abilities, \2 is something you could wipe the floor with. (-12)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is something that you could tear to shreds.', 'Taking stock of its \1 abilities, \2 is something that you could tear to shreds.(-11)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is something that you'd kill quickly.', 'Taking stock of its \1 abilities, \2 is something that you'd kill quickly. (-10)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is a harmless opponent.', 'Taking stock of its \1 abilities, \2 is a harmless opponent.(-9)')
```
Comment (rpherbig) [inline textsubs.lic]: Should this have a space before the parenthetical?

File: textsubs.lic [hunk 2]
```diff
@@ -36,7 +36,32 @@ before_dying do
 end
 
 # defaults
-
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is a creature completely beneath your notice.', 'Taking stock of its \1 abilities, \2 is a creature completely beneath your notice. (-13)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is something you could wipe the floor with.', 'Taking stock of its \1 abilities, \2 is something you could wipe the floor with. (-12)')
+TextSubs.add('^Taking stock of its (offensive|defensive) abilities, (.*) is something that you could tear to shreds.', 'Taking stock of its \1 abilities, \2 is something that you could tear to shreds.(-11)')
```
Comment (rpherbig) [inline textsubs.lic]: Should this have a space before the parenthetical?

**Comment by izawwlgood** [comment]
I'll tweak -11 for consistency. My thinking was

**Pull request merged**